_posts/2017-09-28-our-engineering-culture-methodologies-and-philosophies.md

```diff
@@ -0,0 +1,179 @@
+- https://hackernoon.com/its-a-wonder-some-teams-get-anything-done-64104f59bf5e
+  
+## Scrum  
+Scrum is an Agile methodology that help teams get things done in as productive
+a manner as possible. It allows the business to track project progress
+transparently and get an idea of throughput based on estimation. Scrum allows
+the product team (Product Managers and Engineer) to be able to set expectations
+that stakeholders understand and abide by. While none of us believe in burn down
+charts, for those people that are far removed from the process, it can act as
+a simple flag as to a sprint going well, or falling behind. The most important
+piece of Scrum is the Retrospective. This is an open forum to discuss what went
+well during the sprint, what failed, and what we can do as a team to address
+the issues that led to the failures. Thus far, we have evolved our process with
+our current team to address the issues that have arisen so far, and it has all
+been for the better. You can find our retrospective notes in Confluence.
+
+Here are some articles that speak specifically about Scrum and it's importance
+in organizations.
+- https://martinfowler.com/bliki/FlaccidScrum.html
+- https://www.scrumalliance.org/community/articles/2014/november/top-10-reasons-to-adopt-scrum-in-an-organization
+  
+## Blameless Culture  
+Having a blameless culture is paramount for building a sustainable, long lived
+engineering culture. No matter the project, bugs are going to exist. Whether
+it is due to miscommunication, misinterpretation, speed, lack of tests, etc,
+bugs will always be introduced into new and old systems alike. It is for that
+reason that it is of utmost importance to have a culture of _learning from
+mistakes_ as opposed to vilifying the one individual who made the mistake.
+
+Here are some links that better describe the nuance of a blameless culture and
+it's benefits to a team and it's culture.
+- https://americastestkitchen.github.io/culture/engineering/2015/05/13/the-value-of-postmortems/
+  
+## Technical Debt  
+Every company has technical debt, and it can take many forms. Some are legacy
+projects that engineering teams inherit with it's primary architects long gone.
+Other forms are when a deadline is very tight that some short cuts are taken
+in order to hit deadlines. In either scenario, technical debt has a deliterious
+effect on health of systems, engineering productivity, and more. This topic,
+which has been discussed in length in every possible forum, is of the utmost
+importance due to it's impact.
+
+Here are some links about what technical debt is, and it's place in
+organizations.
+- https://martinfowler.com/bliki/TechnicalDebt.html
+- https://medium.com/modern-agile/who-manages-technical-debt-levels-7e634b0a5974?source=linkShare-fd418bfc22ba-1506544066
+  
+## Code Review  
+Code review is very important in every organization. It acts as a way to
```

Comment:
Three "it acts" in a row. Maybe give your penchant for the oxford comma some love? 😎